charts/tidb-lightning/templates/job.yaml

```diff
@@ -138,6 +148,20 @@ spec:
         persistentVolumeClaim:
           claimName: {{ include "tidb-lightning.name" . }}
       {{ end -}}
+      {{- if and .Values.tlsCluster .Values.tlsCluster.enabled }}
+      - name: lightning-tls
+        secret:
+          secretName: {{ .Values.targetTidbCluster.name }}-lightning-cluster-secret
+      {{- end }}
+      {{- if and .Values.tlsClient .Values.tlsClient.enabled }}
+      - name: tidb-client-tls
+        secret:
+        {{- if and .Values.tlsClient .Values.tlsClient.secretName }}
```

Comment:
`.Values.tlsClient` is already checked in L156.

Reply:
fixed in https://github.com/pingcap/tidb-operator/pull/3598/commits/146540160a947ce7d2cc35e305abbe4c562dab60